org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -389,6 +389,61 @@
      <icon contentType="org.eclipse.wildwebdeveloper.ts" icon="icons/tsEditorIcon.png"/>
    </extension>
 
+   <!-- Java Server Pages (JSP) -->   
+   <extension
+         point="org.eclipse.core.contenttype.contentTypes">
+      <content-type
+            base-type="org.eclipse.wildwebdeveloper.parent"
+            file-extensions="jsp,jspf,tag"
+            id="org.eclipse.wildwebdeveloper.jsp"
+            name="JavaServerPages"
+            priority="normal">
+      </content-type>
+   </extension>
+
+   <extension
```

Comment:
....and get rid of the language server redefinition as the HTML language server would be used (because it's mapped to a parent type)